config/ksql-server.properties

```diff
@@ -80,3 +80,4 @@ compression.type=snappy
 # Uncomment and complete the following to enable KSQL's integration to the Confluent Schema Registry:
 # ksql.schema.registry.url=http://localhost:8081
 
+ksql.rowpartition.rowoffset.enabled = true
```

Comment:
This does not look like it's meant to be checked in.

Reply:
No idea how I missed this and a `FIXME` comment in a previous commit... I definitely got confused and forgot to review this PR before pushing. Sorry for the basic errors recently.